pdns/pdns_recursor.cc

```diff
@@ -2453,30 +2488,32 @@ static void houseKeeping(void *)
 
 static void makeThreadPipes()
 {
-  for(unsigned int n=0; n < g_numThreads; ++n) {
-    struct ThreadPipeSet tps;
+  /* thread 0 is the handler / SNMP, we start at 1 */
+  for(unsigned int n = 1; n <= (g_numWorkerThreads + g_numDistributorThreads); ++n) {
```

Comment:
I'm pondering whether `g_numThreads` should go, because it has not really been the total number of threads for a while, since we have `SNMP` and `protobuf` threads, so it's misleading.